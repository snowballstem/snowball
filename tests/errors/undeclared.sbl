// Regression test for bug reporting names.
externals (stem)
booleans (rb longbool rb)
groupings (rg longgroup rg)
integers (ri longint ri)
strings (rs longstring rs)

define rg 'rg'
define longgroup 'aeiou'
define rg 'xy'

define stem as (
  set longbool
  set xb
  $longint = 1
  $xi = 1
  ri
  $longstring = ''
  $(sizeof xs > 0)
)
