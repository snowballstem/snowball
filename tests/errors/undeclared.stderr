errors/undeclared.sbl:3: 'rb' re-declared
errors/undeclared.sbl:4: 'rg' re-declared
errors/undeclared.sbl:5: 'ri' re-declared
errors/undeclared.sbl:6: 'rs' re-declared
errors/undeclared.sbl:10: 'rg' redefined
errors/undeclared.sbl:14: 'xb' undeclared
errors/undeclared.sbl:16: 'xi' undeclared
errors/undeclared.sbl:17: integer name 'ri' misplaced
errors/undeclared.sbl:19: 'xs' undeclared
